server/chat-plugins/abuse-monitor.ts

```diff
@@ -674,8 +687,35 @@ export const commands: Chat.ChatCommands = {
 			}
 			buf += `<br /><strong>Minimum percent to process:</strong> ${settings.minScore}`;
 			buf += `<br /><strong>Score threshold:</strong> ${settings.threshold}`;
+			buf += `<br /><strong>Threshold increments:</strong>`;
+			const incr = settings.thresholdIncrement;
+			buf += `<br /> &bull; Increases ${incr.amount} every ${incr.turns} turns`;
+			if (incr.minTurns) buf += ` after turn ${incr.minTurns}`;
 			this.sendReplyBox(buf);
 		},
+		ti: 'thresholdincrement',
+		thresholdincrement(target, room, user) {
+			checkAccess(this);
+			if (!toID(target)) {
+				return this.parse(`/help am`);
+			}
+			const [turns, increment, min] = Utils.splitFirst(target, ',', 2).map(r => parseInt(toID(r)));
+			if (isNaN(turns) || turns < 0) {
+				return this.errorReply(`Turns must be a number above 0.`);
+			}
+			if (isNaN(increment) || increment < 0) {
+				return this.errorReply(`The increment must be a number above 0.`);
+			}
+			settings.thresholdIncrement = {amount: increment, turns, minTurns: min || 0};
```

Comment:
Why set `minTurns` to 0 if there's no value specified? Shouldn't it just not be included (a la `if (min) settings.thresholdIncrement.min = min`)?
Also, what if someone specifies a non-numeric value for `min` in the command?

Reply:
Then why type it as an optional property at all?

Reply:
> Also, what if someone specifies a non-numeric value for min in the command?

```ts
> NaN || 0
0
```

Reply:
> Then why type it as an optional property at all?

Doesn't really matter either way, idt. I had originally planned to make it not defined, but then realized this made the code a lot simpler.

Reply:
If someone makes a typo (say "3p" instead of "30"), it should not silently default to 0 — it should display an error message so that they realize their mistake and can retype it as "30" (or whatever they meant).

Reply:
OK.

Reply:
> > Then why type it as an optional property at all?
> 
> Doesn't really matter either way, idt. I had originally planned to make it not defined, but then realized this made the code a lot simpler.

I don't like the idea of having two ways to say "no minimum turn value" - both `undefined` and `0` work. IMO the former has clearer typing while the latter has simpler (thus more readable) code. Either is fine, but you should pick one and only use it.